builder/builder.js

```diff
@@ -28,14 +40,49 @@ window.addEventListener('load', async() => {
         displayTown(town);
     }
 });
+//land dropdown
+    //pull DOM, async/change, const booger=awaitFetchFunc(DOM.value), call displayTown(booger)
+landDropdown.addEventListener('change', async() => {
+    const newLand = await updateLand(landDropdown.value);
+
+    displayTown(newLand);
+});
+//castle dropdown
+castleDropdown.addEventListener('change', async() => {
+    const newCastle = await updateCastle(castleDropdown.value);
+
+    displayTown(newCastle);
+});
+//creature dropdown
+creatureDropdown.addEventListener('change', async() => {
+    const newCreature = await updateCreature(creatureDropdown.value);
+
+    displayTown(newCreature);
+});
+//name form & button
+nameForm.addEventListener('submit', async(e) => {
+    e.preventDefault();
+
+    const data = new FormData(nameForm);
+
+    const name = data.get('name');
+
+    const updatedTown = await updateName(name);
+
+    displayTown(updatedTown);
+
+    nameForm.reset();
+});
```

Comment:
You need to make an event listener for the slogans, which explains why they are not displaying on your page.